clippy_lints/src/lib.rs

```diff
@@ -570,6 +571,7 @@ pub fn register_plugins(reg: &mut rustc_plugin::Registry<'_>, conf: &Conf) {
     reg.register_late_lint_pass(box types::RefToMut);
     reg.register_late_lint_pass(box assertions_on_constants::AssertionsOnConstants);
     reg.register_late_lint_pass(box missing_const_for_fn::MissingConstForFn);
+    reg.register_late_lint_pass(box transmuting_null::TRANSMUTING_NULL);
```

Comment:
Do not use all upper case here, it should be `box transmuting_null::TransmutingNull`.

Reply:
Thank you!